Source/dom/style/Text.js

```diff
@@ -175,7 +175,10 @@ export function defineTextStyleProperties(Style) {
         return undefined
       }
 
-      const raw = paragraphStyle.alignment()
+      // eslint-disable-next-line
```

Comment:
Another option is to add `MSTextAlignmentConverter` to the `.eslintrc`, which seems to be how Sketch's types are handled in the project.